### PR TITLE
Download agate-apps at first boot and run communication-server at sta…

### PR DESCRIPTION
**Pull request by andre-rosa**
…rtup

### CODE REVIEW

File: recipes-apps/agate-apps/agate-apps.bb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+"
+
+PR = "r0"
+SRCREV = "${AUTOREV}"
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+S="${WORKDIR}"
+
+SRC_URI += " \
+    file://20-agate-apps-install.sh \
+    file://agate-agl-app.wgt \
```
Comment (jdapena) [inline recipes-apps/agate-apps/agate-apps.bb:15]: we should stop doing this, and build the full application from yocto recipe.

File: recipes-apps/agate-apps/files/20-agate-apps-install.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  exit 1
+}
+
+cd /home/root/app-data || exit 1
+git clone https://github.com/enactjs/agate-apps.git
+cd agate-apps || exit 1
+run_cmd_flaggling_execution "npm install -g @enact/cli" cli
+for PKG in console copilot communication-server components; do
+  (cd $PKG || exit 1; run_cmd_flaggling_execution "npm install" $PKG.install)&
+done
```
Comment (jdapena) [inline recipes-apps/agate-apps/files/20-agate-apps-install.sh:34]: Defiinitely this is for this layer, we cnanot land this in ofifcial layers definitely.

**Review by jdapena — APPROVED** [review]
This is valid in meta-agl-lge, but never landable in official AGL. Anyway, let's think about how to rework the apps package later.

File: recipes-apps/agate-apps/agate-apps.bb (hunk 1 above)
Comment (andre-rosa) [reply recipes-apps/agate-apps/agate-apps.bb:15]: Yes, soon once wam lands into guppy we should build and package agate apps properly from recipe.

File: recipes-apps/agate-apps/files/20-agate-apps-install.sh (hunk 2 above)
Comment (andre-rosa) [reply recipes-apps/agate-apps/files/20-agate-apps-install.sh:34]: Definitely! ;-) It's just for demo and quick prototyping purposes.

**Pull request merged**